images/dcos/Dockerfile

```diff
@@ -1,25 +1,32 @@
-FROM busybox:ubuntu-14.04
+FROM alpine:3.2
```

Comment:
I noticed differences already, I think in connection with multiple nameservers, one returning an error. But I think this was due to the nginx compiled against ulibc, not glibc. The image has both, the go binaries link against glibc.